test/test_torch.py

```diff
@@ -16656,6 +16656,39 @@ def test_dot(self, device, dtype):
         torch.dot(v1, v2, out=out)
         self.assertEqual(res1, out)
 
+    @dtypes(torch.float, torch.double, torch.cfloat, torch.cdouble)
+    def test_vdot(self, device, dtype):
+        def compare_with_numpy_bin_op(torch_fn, np_fn, x, y, relaxed_tolerance=False):
+            if self.device_type == 'cuda':
+                y_np = y.cpu().numpy()
+            else:
+                y_np = y.numpy()
+
+            rtol, atol = None, None
```

Comment:
Instead of setting the atol/rtol directly I think you can just the @precisionOverride decorator. To make the tests pass I think you'll need:

```
@precisionOverride({torch.cfloat: 1e-4})
```